pulumi/backuprunner/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "backuprunner",
+  "version": "1.0.0",
+  "main": "index.js",
+  "license": "MIT",
+  "type": "module",
+  "scripts": {
+    "start": "ts-node-esm dist/index.js",
+    "build": "npx tsc",
+    "dev": "ts-node-esm src/index.ts"
+  },
+  "dependencies": {
+    "@kubernetes/client-node": "^0.18.1",
+    "@types/figlet": "^1.5.5",
+    "@types/node": "^18.14.6",
+    "commander": "^10.0.0",
+    "figlet": "^1.5.2",
+    "ts-node": "^10.9.1",
+    "typescript": "^4.9.5"
```

Comment:
would like to keep these in sync with `node/package.json` for now. (we can probably upgrade `node/package.json`) to match so we are current